migration/validator-registration.md

```diff
@@ -0,0 +1,95 @@
+# Registration Guide
+
+This document guides users through the registration process to become one of the first validators on the Nimiq PoS chain. It is part of the overall documentation for the transition from PoW to PoS. For registering a validator post-transition, refer to the [Becoming a Validator](/build/set-up-your-own-node/becoming-a-validator.md) guide. For more technical details on the migration, see [Migration Technical Details](migration-technical-details).
+
+## Validator Registration Tool
+
+This guide focuses on **Phase 1: Validator Registration** of the migration process, which takes place between **September 12 and October 6**. During this phase, you will create and register your validator on the PoW Mainnet, preparing it for the transition to PoS.
+
+Nimiq has developed a **Validator Registration Tool** specifically for this process. By using this tool, validators pre-register within the PoW chain, ensuring they are ready for the PoS transition. The tool operates in two modes:
+
+- **Key and Address Generation**: Generates the validator's keys and corresponding address
+- **Validator Registration**: Registers the validator by sending the necessary transactions on the PoW chain
+
+| **Phase** | **Start** | **End** |
+| --- | --- | --- |
+| **Validator Registration** | **12th September** | **6th October** |
+| Pre-Stake Phase | 7th October | 10th November |
+| Validator Activation | 19th November | - |
+
+The steps outlined in this guide are only applicable during the **Validator Registration Phase**.
+
+The **Activation Phase** begins on **November 19**, and we recommend running the activation tool **before** this date to allow your node to download the database early. Learn more about the activation process [here](#what-s-next).
```

Comment:
I'd remove this line from this guide since this guide is talking about the validator registration phase.